docs/data-management/post-session.md

```diff
@@ -220,21 +236,31 @@ To support backward compatibility (and some extra, currently unsupported feature
     │   ├── sub-001_ses-024_task-rest_dir-AP_echo-4_part-mag_bold.nii.gz
     │   ├── sub-001_ses-024_task-rest_dir-AP_echo-4_part-phase_bold.json
     │   ├── sub-001_ses-024_task-rest_dir-AP_echo-4_part-phase_bold.nii.gz
-    │   ├── sub-001_ses-024_task-rest_dir-AP_events.json
-    │   └── sub-001_ses-024_task-rest_dir-AP_events.tsv
+    │   ├── sub-001_ses-024_task-rest_dir-AP_recording-cardiac_physio.json
+    │   ├── sub-001_ses-024_task-rest_dir-AP_recording-cardiac_physio.tsv.gz
+    │   ├── sub-001_ses-024_task-rest_dir-AP_recording-respiratory_physio.json
+    │   ├── sub-001_ses-024_task-rest_dir-AP_recording-respiratory_physio.tsv.gz
+    │   ├── sub-001_ses-024_task-rest_dir-AP_stim.json
+    │   └── sub-001_ses-024_task-rest_dir-AP_stim.tsv.gz
     └── sub-001_ses-024_scans.tsv
     ```
 
-### Convert physiological recordings into BIDS (in-house)
+### Generate BIDS' *events* files
```

Comment:
Why the order of this has changed? Git is confused after postponing the events files generation after physio. Can you roll this back?